api/reporting/docs/cohort_definition_schema.json

```diff
@@ -0,0 +1,201 @@
+              "type": "string"
+            }
+          ]
+        }
+      },
+      "required": ["name", "operator", "operands"]
+    },
+    "modifier": {
+      "description": "A modifier can be age at event, number of occurrences or visit type",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the modifier",
+          "type": "string"
+        },
+        "operator": {
+          "description": "An operator, ex: =, !=, <, <=, >, >=, between, like, in",
+          "type": "string"
+        },
+        "operands": {
+          "description": "A collection of operands",
+          "type": "array",
+          "items": [
+            {
+              "description": "An operand",
+              "type": "integer"
+            }
+          ]
+        }
+      },
+      "required": ["name", "operator", "operands"]
+    },
+    "search_group": {
+      "description": "A search group is a collection of concepts which can be 'OR'ed together",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "A unique identifier for the search group",
+          "type": "string"
+        },
+        "items": {
+          "description": "A collection of search group items",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/search_group_item"
+            }
+          ]
+        },
+        "temporal": {
+          "description": "Is the search group temporal(relating groups within a time frame). ",
+          "type": "boolean"
+        }
+      },
+      "required": ["id", "items", "temporal"]
+    },
+    "search_group_item": {
+      "description": "A search group item is a collection of concepts of the same domain which can be 'OR'ed together",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The unique identifier for the search group item",
+          "type": "string"
+        },
+        "type": {
+          "description": "The description for this search group item",
+          "type": "string"
+        },
+        "searchParameters": {
+          "description": "A collection search parameters",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/search_parameter"
+            }
+          ]
+        },
+        "modifiers": {
+          "description": "A collection of modifiers that refine the search",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/modifier"
+            }
+          ]
+        },
+        "temporalGroup": {
+          "description": "Temporal group index for the search group item",
+          "type": "integer"
+        }
+      },
+      "required": ["id", "type", "searchParameters", "modifiers"]
+    },
+    "search_parameter": {
+      "description": "A search parameter contains metadata about a specific concept",
```

Comment:
please give an example or two of a search parameter.

Reply:
Done